pymc3/sampling.py

```diff
@@ -91,11 +93,13 @@
 )
 
 ArrayLike = Union[np.ndarray, List[float]]
+PointType = Dict[str, np.ndarray]
+PointList = List[PointType]
 
 _log = logging.getLogger("pymc3")
 
 
-def instantiate_steppers(model, steps, selected_steps, step_kwargs=None):
+def instantiate_steppers(_model, steps, selected_steps, step_kwargs=None):
```

Comment:
I'm not sure what to do about this, since I made this change to quash a lint warning and reflect the fact that `model` is bound  but not read.  I didn't change the API at all, just updated the docstring and changed the variable name to reflect existing practice.

If you'd prefer, I'm happy to just undo this, but I'm not sure why it would be a breaking change.

Reply:
Oh I though you wanted to deprecate the argument in the long run?
I don't see how re-naming it helps. Also, theoretically, someone _could_ pass it as `instantiate_steppers(model=...)` and that would break.

Reply:
I used the underscore to tell pylint not to complain about the unread variable...

Reply:
If you prefer, I could put the name back and add a pylint disable comment

Reply:
sounds fine